magefile.go

```diff
@@ -45,23 +173,17 @@ func Check() {
 
 // CheckLicense checks the license headers
 func CheckLicense() error {
-	mg.Deps(InstallLicenser)
+	mg.Deps(mage.InstallGoLicenser)
 
 	return gotool.Licenser(
 		gotool.Licenser.License("Elastic"),
 		gotool.Licenser.Check(),
 	)
 }
 
-// InstallLicenser installs the licenser in the current environment
-func InstallLicenser() error {
-	return gotool.Install(gotool.Install.Package(goLicenserRepo))
-}
-
-// CheckLicense checks the license headers
+// License should generate the license headers
```

Comment:
[Suggestion]
Personally, I find better to have a function description to be assertive, it does either A or B, not it "should" do something.
```suggestion
// License generates the license headers or returns an error
```